### DISCUSSION TITLE
install netbox 3.1.5 on debian 11 Server Error

### DISCUSSION THREAD

**Discussion by akmorp** [netbox-community/netbox#8330] [Q&A] [answered]
debian 11, python 3.9.2, postgresql 14 last from apt repo, redis 6.2.6 last from apt repo
On step test the application from docs
python3 manage.py runserver 0.0.0.0:8000 --insecure

in browser:
-----------------------------
Server Error

There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'redis.exceptions.ConnectionError'>

Error 111 connecting to localhost:6379. Connection refused.

Python version: 3.9.2
NetBox version: 3.1.5

If further assistance is required, please post to the NetBox discussion forum on GitHub.
----------------------------------------

in console:
---------------------
/opt/netbox/venv/lib/python3.9/site-packages/django/views/debug.py:420: ExceptionCycleWarning: Cycle in the exception chain detected: exception 'Error 111 connecting to localhost:6379. Connection refused.' encountered again.
  warnings.warn(
[12/Jan/2022 14:00:37] "GET / HTTP/1.1" 500 1626
[12/Jan/2022 14:00:37] "GET /static/netbox-light.css HTTP/1.1" 200 232256
[12/Jan/2022 14:00:37] "GET /favicon.ico HTTP/1.1" 500 1626

----------------------

**Comment by markkuleinio** [comment, marked as answer]
Is your redis running?

sudo systemctl status redis-server
sudo systemctl start redis-server
sudo systemctl enable redis-server

**Reply by akmorp** [reply]
Thank you.
